src/pages/index.js

```diff
@@ -15,9 +17,16 @@ export default function Home() {
         <meta name="description" content="Generated by create next app" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <main className={styles.main}></main>
-      <FooterBar />
+
+      </Head> 
+      
+      <main>
+        
+      </main>
```

Comment:
@Sheyitofunmi If `<FooterBar>` is included in the `main` branch, please do not remove it from `index.js`, or else after I merge your branch, everyone will lose the `<Footer>` component in all their local apps.